site/content/communities/slack.md

```diff
@@ -131,7 +136,7 @@ If you'd like to participate in the conversation about the future of the communi
 - `#website-github`
   - Automated activity feed from https://github.com/techqueria/website
 
-There's also the `#twitter` channel (Techqueria's Twitter feeds) and `#websit
+There's also the `#twitter` channel (Techqueria's Twitter feeds) and `#website`
```

Comment:
Let's add to the list instead of sentence at bottom

Reply:
Thanks for typo fix!

Reply:
#twitter and #website channels already listed, want me to just take them out? @fvcproductions

Reply:
Yeah that works!